tests/dummy/app/helpers/check-data-type.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function checkDataType(resource) {
```

Comment:
This seems to be a validation function?

For internal use in the dummy app that's probably fine. 

As we build components, we will want to consider using an established prebuilt form validation library for this purpose rather than writing our own.